Packs/DelineaSS/pack_metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Delinea Secret Server",
+    "description": "Secure privileges for service, application, root and administrator accounts across your enterprise.",
+    "support": "partner",
+    "currentVersion": "2.0.0",

```

Comment:
Please change to `1.0.0`, as our build process expects new packs to always start with `1.0.0`.